src/pages/search/advanced_search/PublisherFilter.tsx

```diff
@@ -0,0 +1,90 @@
+import { Autocomplete, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router';
+import { searchForPublishers } from '../../../api/publicationChannelApi';
+import { ResultParam } from '../../../api/searchApi';
+import { AutocompleteTextField } from '../../../components/AutocompleteTextField';
+import { Publisher } from '../../../types/registration.types';
+import { useDebounce } from '../../../utils/hooks/useDebounce';
+
+export const PublisherFilter = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const publisherParam = searchParams.get(ResultParam.Publisher);
+  const [publisherQuery, setPublisherQuery] = useState('');
+  const debouncedQuery = useDebounce(publisherQuery);
+
+  const publisherOptionsQuery = useQuery({
+    queryKey: ['publisherSearch', debouncedQuery],
+    enabled: debouncedQuery.length > 3 && debouncedQuery === publisherQuery,
+    queryFn: () => searchForPublishers(debouncedQuery, '2023'),
+    meta: { errorMessage: t('feedback.error.get_publishers') },
+  });
+
+  const publisherList = publisherOptionsQuery.data?.hits ?? [];
+
+  const selectedPublisherQuery = useQuery({
+    enabled: !!publisherParam,
+    queryKey: [publisherParam],
+    queryFn: () => searchForPublishers(publisherParam ?? '', '2023'),
+    meta: { errorMessage: t('feedback.error.get_publisher') },
+    staleTime: Infinity,
+  });
+
+  useEffect(() => {
+    if (selectedPublisherQuery.data) {
+      setPublisherQuery(selectedPublisherQuery.data.hits[0].name);
+    }
+  }, [selectedPublisherQuery.data]);
```

Comment:
skjønner ikke denne helt.. 🤔 Hvorfor?
```suggestion
```

Reply:
For å få satt korrekt `inputValue` på page refresh. Mulig jeg har misforstått hvordan dette henger sammen, men tanken var å forhinfre controlled -> uncontrolled

Reply:
Mulig jeg misser noe her altså, men ser ut for meg som en ekstra setState (og rerender) uten at det har noen effekt, verken synlig eller warnings 🤔 
Å sette søketerm lik oppslaget virker ikke åpenbart for meg heller